packages/app-store/routing-forms/lib/jsonLogicOverrides.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import jsonLogic from "json-logic-js";
+
+// converts input to lowercase if string
+function normalize(input: any) {
+  return typeof input === "string" ? input.toLowerCase() : input;
+}
+
+jsonLogic.add_operation("==", function (a: any, b: any) {
+  return normalize(a) == normalize(b);
+});
+
+jsonLogic.add_operation("===", function (a: any, b: any) {
+  return normalize(a) === normalize(b);
+});
+
+jsonLogic.add_operation("!==", function (a: any, b: any) {
+  return normalize(a) !== normalize(b);
+});
+
+jsonLogic.add_operation("!=", function (a: any, b: any) {
+  return normalize(a) != normalize(b);
+});
+
+jsonLogic.add_operation("in", function (a: any, b: any) {
+  const first = normalize(a);
+  const second = normalize(b);
+  if (!second || typeof second.indexOf === "undefined") return false;
```

Comment:
What would be the case when second.indexOf is undefined. As per my testing second is either an array of string or string. I couldn't find such a case.

Reply:
The "in" function was copied over from the jsonLogic github repository. 
My intention was to not change anything besides normalizing the input.

I assumed it was to cater for a corner case but will review and decide whether
to modify it with the expectation of second being string or string[] as you found
out from your tests.